src/Utilities/Constants.f90

```diff
@@ -23,7 +23,7 @@ module ConstantsModule
   integer(I4B), parameter :: LENOBSNAME = 40
   integer(I4B), parameter :: LENOBSTYPE = 30
   integer(I4B), parameter :: LENTIMESERIESNAME = LENOBSNAME
-  integer(I4B), parameter :: LENTIMESERIESTEXT = 12
+  integer(I4B), parameter :: LENTIMESERIESTEXT = 16
```

Comment:
FYI, this is probably the most visible change, previously effectively setting `tslink%Text = 'CONCENTRATIO'` in src/Model/GroundWaterTransport/gwt1cnc1.f90:480.

Are there any hidden side-effects of this change? E.g. formatting that assumes a width, or binary file records?

Reply:
The alternative is to keep the length at 12, and abbreviate the one truncation instance. There are plenty of other `tslink%Text` examples that use abbreviated forms, e.g. see `git grep -i tslink%text`